tests/integration/helpers.py

```diff
@@ -36,9 +36,11 @@ async def get_password(ops_test: OpsTest, app=APP_NAME) -> str:
     unit_name = ops_test.model.applications[app].units[0].name
     unit_id = unit_name.split("/")[1]
 
-    action = await ops_test.model.units.get(f"{app}/{unit_id}").run_action("get-password")
+    action = await ops_test.model.units.get(f"{app}/{unit_id}").run_action(
+        "get-password", **{"username": username}
+    )
     action = await action.wait()
-    return action.results["operator-password"]
+    return action.results[f"{username}-password"]
```

Comment:
lets simplify parsing of results to 3rd-party charms.
```suggestion
    return action.results["password"]
```
can we check what keys other charms have?

Reply:
Yes will do. 

[This is the info](https://docs.google.com/spreadsheets/d/1mb_cN92iEP0UEJzh7RHZXwJlYd0Yi_y7N6Frw4z3rUs/edit#gid=0) I've collected on how the other charms manage passwords

Reply:
@delgod, should we align with the other database charm developers to change the other charms as well?

Reply:
@marceloneppel yes please